dss/collections/owner_lookup.py

```diff
@@ -0,0 +1,49 @@
+import os
+
+from dss import dynamodb  # type: ignore
+
+
+collection_db_table = f"dss-collections-db-{os.environ['DSS_DEPLOYMENT_STAGE']}"
+
+
+def put_collection(owner: str, versioned_uuid: str, permission_level: str = 'owner'):
```

Comment:
Please change `versioned_uuid` to `collection_fqid`

This should be applied throughout this PR.